platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/WebSocket.js

```diff
@@ -69,10 +85,10 @@ export default class WebSocket {
     my.onSocketError(this._onError)
 
     this._onClose = (res) => {
-      this.readyState = WebSocket.CLOSED
-      this._triggerEvent('close')
-      this._removeAllSocketListenr()
+      this.readyState = WebSocket.CLOSED;
       this._decreaseCount();
+      this._removeAllSocketListener();
+      this._triggerEvent('close');
```

Comment:
Does the order of removing listeners and firing close event matter?

Reply:
Due to the singleton nature of the class, failing to call decrease before callback will result in the creation of a new WebSocket instance within the callback, which will fail.